src/graphql/export.rs

```diff
@@ -2336,7 +2336,7 @@ fn parse_key(key: &[u8]) -> anyhow::Result<(Cow<str>, i64)> {
     Err(anyhow!("Invalid key"))
 }
 
-fn to_vec_string<T>(vec: &Vec<T>) -> Vec<String>
+fn to_vec_string<T>(vec: &[T]) -> Vec<String>
```

Comment:
This change is to satisfy latest clippy version, accompanied by CI server rust version update to 1.76.0.